src/main/scala/mesosphere/marathon/core/heartbeat/HeartbeatActor.scala

```diff
@@ -0,0 +1,97 @@
+package mesosphere.marathon.core.heartbeat
+
+import akka.actor._
+import scala.concurrent.duration._
+
+// HeartbeatActor monitors the heartbeat of some process and executes handlers for various conditions.
```

Comment:
Is there a reason why you didn't use scaladoc syntax? not that we're consistent with that ..